# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../webkit/icu.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Index: WebCore/platform/text/TextBreakIterator.
===================================================================
--- ./WebCore/platform/text/TextBreakIterator.h.orig 2009-12-21 01:32:31.000000000 -0500
+++ ./WebCore/platform/text/TextBreakIterator.h 2009-12-21 01:32:22.000000000 -0500
@@ -26,7 +26,7 @@
 
 namespace WebCore {
 
-    class TextBreakIterator;
+    typedef struct UBreakIterator TextBreakIterator;
 
     // Note: The returned iterator is good only until you get another iterator.

